packages/expo-doctor/src/index.ts

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env node
+import { constants, promises as fs } from 'fs';
+import path from 'path';
+
+import { actionAsync } from './doctor';
+
+const packageJson = () => require('../package.json');
+
+async function run() {
+  const args = process.argv.slice(2);
+
+  if (args.some(arg => ['-V', '-v', '--version'].includes(arg))) {
```

Comment:
For consistency, our other CLIs (@expo/cli and eas-cli) only support `-v` and `--version`, not `-V`
```suggestion
  if (args.some(arg => ['-v', '--version'].includes(arg))) {
```